proto/ord_schema.proto

```diff
@@ -297,104 +314,120 @@ message ReactionConditions {
 }
 
 message TemperatureConditions {
-  enum TemperatureControl {
-    TEMPERATURE_CONTROL_UNKNOWN = 0;
-    AMBIENT = 1;
-    OIL_BATH = 2;
-    WATER_BATH = 3;
-    SAND_BATH = 4;
-    ICE_BATH = 5;
-    DRY_ALUMINUM_PLATE = 6;
-    MICROWAVE = 7;
-    DRY_ICE_BATH = 8;
-    AIR_FAN = 9;
-    LIQUID_NITROGEN = 10;
+  message TemperatureControl {
+    enum TemperatureControlType {
+      UNSPECIFIED = 0;
+      AMBIENT = 1;
+      OIL_BATH = 2;
+      WATER_BATH = 3;
+      SAND_BATH = 4;
+      ICE_BATH = 5;
+      DRY_ALUMINUM_PLATE = 6;
+      MICROWAVE = 7;
+      DRY_ICE_BATH = 8;
+      AIR_FAN = 9;
+      LIQUID_NITROGEN = 10;
+    }
+    oneof kind {
+      TemperatureControlType type = 1;
+      string custom = 2;
+    }
   }
   TemperatureControl temperature_control = 1;
-  // Used to describe an UNKNOWN temperature control.
-  string temperature_control_custom = 2;
-  Temperature setpoint = 3;
+  Temperature setpoint = 2;
   message Measurement {
+    enum MeasurementType {
```

Comment:
I moved `MeasurementType` into `Measurement`, but I'm not sure I should have. This requires the type to be set for each measurement, which allows multiple types but could be a burden if we only expect one type. What do you think?

Reply:
Keeping the type inside ```Measurement``` certainly increases our flexibility in case people do use multiple methods of measuring the temperature or pressure. Let's keep it inside